bin/p2-watch/main.go

```diff
@@ -122,3 +134,39 @@ func watchPodClusters(client consulutil.ConsulClient, applicator labels.Applicat
 		log.Fatalf("error watching pod cluster: %v", err)
 	}
 }
+
+func watchHealth(service string, client consulutil.ConsulClient) {
+	hc := checker.NewConsulHealthChecker(client)
+	healthResults := make(chan map[types.NodeName]health.Result)
+	errCh := make(chan error)
+	quitCh := make(chan struct{})
+	go hc.WatchService(*healthService, healthResults, errCh, quitCh)
+
+	go func() {
+		signalCh := make(chan os.Signal, 2)
+		signal.Notify(signalCh, syscall.SIGTERM, os.Interrupt)
+		received := <-signalCh
+		log.Printf("[WARN] Received %v, shutting down", received)
+		close(quitCh)
+	}()
+
+	var hr map[types.NodeName]health.Result
+	var sortedHealthResults []health.Result
+	var err error
+	for {
+		select {
+		case hr = <-healthResults:
+			for _, r := range hr {
+				fmt.Printf("%s %s\n", r.Node.String(), r.Status)
+				sortedHealthResults = append(sortedHealthResults, r)
```

Comment:
confused. You don't seem to do anything with these sortedHealthResults...

Nor do you appear to actually sort anything.